assets/section-main-product.css

```diff
@@ -540,8 +569,14 @@ a.product__text {
 }
 
 .product__media-toggle {
-  background-color: transparent;
+  display: flex;
   border: none;
+  background: transparent;
```

Comment:
should we keep `background-color` here ? I think it's better since we're only changing the background color.

Reply:
Since background is also referring to other properties that aren't tweaked here, I think `backrgound-color` is the way to go no?